.changeset/vast-goats-dream.md

```diff
@@ -0,0 +1,25 @@
+---
+'@reown/appkit': patch
+'@reown/appkit-adapter-bitcoin': patch
+'@reown/appkit-adapter-ethers': patch
+'@reown/appkit-adapter-ethers5': patch
+'@reown/appkit-adapter-solana': patch
+'@reown/appkit-adapter-wagmi': patch
+'@reown/appkit-utils': patch
+'@reown/appkit-cdn': patch
+'@reown/appkit-cli': patch
+'@reown/appkit-common': patch
+'@reown/appkit-controllers': patch
+'@reown/appkit-core': patch
+'@reown/appkit-experimental': patch
+'@reown/appkit-pay': patch
+'@reown/appkit-polyfills': patch
+'@reown/appkit-scaffold-ui': patch
+'@reown/appkit-siwe': patch
+'@reown/appkit-siwx': patch
+'@reown/appkit-ui': patch
+'@reown/appkit-wallet': patch
+'@reown/appkit-wallet-button': patch
```

Comment:
nitpick: this is `minor`? is there a chance apps' implementations relies on how this currently works? if so `minor` else `patch`

Reply:
Good point. I think this is still patch. In terms of the values that `walletInfo` gets over time after page refresh:

- First render: `undefined`
- Second render: `walletConnect`
- Third render: `Safe{Wallet}` (assuming connected with safe)

After this PR:

- First render: `undefined`
- Second render: `Safe{Wallet}` (assuming connected with safe)

If somebody expecting `walletInfo.name` to be `walletConnect`, they already know that this immediately turns to another value. This means they probably will use this only for watching the value to trigger something, not to render something to the screen depending on this value.

But this is a good point and I'd like to ask to the @reown-com/appkit-web-team wdyt?

Reply:
I think this is still a patch as the end behavior is still the same, avoiding a glitchy behavior